internal/cli/pkg/cluster/create.go

```diff
@@ -0,0 +1,331 @@
+/*
```

Comment:
The package `pkg` is redundant which can be removed or renamed to another meaningful one.